Include/internal/pycore_pyatomic_ft_wrappers.h

```diff
@@ -59,6 +59,8 @@ extern "C" {
     _Py_atomic_store_uint16_relaxed(&value, new_value)
 #define FT_ATOMIC_STORE_UINT32_RELAXED(value, new_value) \
     _Py_atomic_store_uint32_relaxed(&value, new_value)
+#define FT_ATOMIC_EXCHANGE_PTR(value, new_value) \
```

Comment:
I'm not using this now, should I leave it in?

Reply:
Let's remove it then.